Makefile

```diff
@@ -72,6 +72,7 @@ appstore:
 	--exclude=/.scrutinizer.yml \
 	--exclude=/.travis.yml \
 	--exclude=/check-vuejs-builds.sh \
+	--exclude=/cypress \
```

Comment:
We should also add `cypress.config.js` here. I'll push a follow-up commit to the PR.